test/run-tests

```diff
@@ -353,6 +354,44 @@ class Task:
                 f"in image {self.image['name']}"
             )
 
+    def is_role_distro_supported(self):
+        """
+        Return True if image is supported. Otherwise False.
+        """
+        distro = self.image["name"].split("-")[0]
+        version = self.image["name"].split("-")[1]
```

Comment:
```python
distro, version = self.image["name"].split("-")
```

Reply:
This also has the added benefit that it will throw an exception if the format of name is not in the expected `DISTRO-VERSION` format

Reply:
OK. Fixed in #6dc7afd.